frontend/src/scenes/authentication/loginLogic.ts

```diff
@@ -43,6 +54,11 @@ export const loginLogic = kea<loginLogicType<AuthenticateResponseType>>({
                 window.location.href = afterLoginRedirect()
             }
         },
+        precheckSuccess: ({ precheckResponse }) => {
+            if (precheckResponse?.sso_enforcement) {
+                window.location.href = `/login/${precheckResponse.sso_enforcement}/`
```

Comment:
Not sure about this UX @clarkus, is redirecting automatically okay or should we change the "Login" button to a "Login with X" and let the user click it?

Reply:
Right now it feels quite abrupt to go from one login to the SSO one, perhaps a transition here or at least an indication that the user is now logging in with SSO could help?

Reply:
Makes sense @liyiy, thanks for the feedback! Please see updated description with details.

Reply:
lovely